src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -578,7 +578,7 @@ h2.location a {
 	position: relative;
 }
 
-.example-wrap > pre.line-number {
+.example-wrap > pre.example-line-numbers {
```

Comment:
We could even simplify such rules since there is no conflict between the doc and source pages.

Reply:
Yes because I don't think it's needed anymore.